models/articles.models.js

```diff
@@ -1,14 +1,28 @@
 const db = require("../db/connection");
 
-exports.fetchAllArticles = () => {
-  return db.query(`
+exports.fetchAllArticles = (filterBy) => {
+  let queryString = `
   SELECT articles.author, articles.title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url, COUNT(comments) AS comment_count
   FROM articles
   LEFT JOIN comments
-  ON articles.article_id = comments.article_id
-  GROUP BY articles.article_id
-  ORDER BY created_at DESC
-  `);
+  ON articles.article_id = comments.article_id`;
+
+  if (filterBy) {
+    const validFilterQueries = ["mitch", "cats", "paper"];
```

Comment:
you have to check the topic exist in the db rather than hard coding.  The api with your production data would always give a 404 with these hardcoded values.